appinfo/routes.php

```diff
@@ -99,14 +102,14 @@
 		['name' => 'publicPageTrash#restore', 'url' => '/_api/p/{token}/_pages/trash/{id}', 'verb' => 'PATCH',
 			'requirements' => ['id' => '\d+']],
 
-		// default public route (Vue.js frontend)
+		// Vue.js router public route (Vue.js frontend)
 		['name' => 'publicStart#publicIndex', 'url' => '/p/{token}/{path}', 'verb' => 'GET',
 			'requirements' => ['path' => '.*'],	'defaults' => ['path' => '']],
 
-		// default route (Vue.js frontend)
-		['name' => 'start#index', 'url' => '/{path}', 'verb' => 'GET',
+		// Vue.js router route (Vue.js frontend)
+		['name' => 'start#indexPath', 'url' => '/{path}', 'verb' => 'GET',
```

Comment:
Maybe move this up to the other route of StartController to have them all in one place?

Reply:
Since these are basically catch-all routes, they need to come last. We could move the route without path param down to have all start controller routes together. Probably a good idea :+1:

Reply:
Ah I see, wasn't aware of that.